src/boot/efi/drivers.c

```diff
@@ -50,7 +50,7 @@ static EFI_STATUS load_one_driver(
         return EFI_SUCCESS;
 }
 
-static EFI_STATUS reconnect(void) {
+EFI_STATUS reconnect(void) {
```

Comment:
hmm, if we export this, can we give this a bit more descriptive name, please? short, brief names within a local context is fine, but if we expose this to other code, then let's try to find a better name. `reconnect_all_drivers()` or `reconnect_all_devices()` or so?